Yank/yank.py

```diff
@@ -131,7 +131,7 @@ class Yank(object):
         'mc_displacement_sigma': 10.0 * unit.angstroms
     }
 
-    def __init__(self, store_directory, mpicomm=None, **kwargs):
+    def __init__(self, store_directory, mpicomm=None, platform=None, **kwargs):
```

Comment:
This should really be thrown in with the `**kwargs`` and not carried around as its own call